.gitignore

```diff
@@ -13,6 +13,9 @@ Makefile.in
 *.lo
 *.a
 *.la
+.cproject
+.project
+.settings/language.settings.xml
 
```

Comment:
I'm using Eclipse and it adds these files. Not sure if it's better, I'm just more accustomed to it :)